packages/scripts/src/analyzeMessages.ts

```diff
@@ -98,7 +98,7 @@ const analyzeMessages = async ({ db }: { db: Db }) => {
         embedding: 0,
       },
     },
-    { $sort: { conversationId: 1, createdAt: 1 } },
+    { $sort: { convoId: 1, createdAt: 1 } },
```

Comment:
why this change?

Reply:
Fixes a bug, note that the field is projected to "convoId" not "conversationId" above